kastenhq/kopia
# Add error for token expiration.

When an AWS session token expires, kopia should treat token expiration errors as non retriable errors. This will ensure that transfers fail fast and the error is reported back to the user sooner. 


Testing:

- Need to setup an app with a PVC with reasonable amount of data that would ensure that transfers last for atleast 15 minutes . The minimum session token duration is 15 minutes.
- Set parallelism to lowest value of 1 to slow down the transfers. 
- Verify that transfers fail fast on encountering the "The provided token has expired" error .
